Phonebook.Frontend/src/styles.scss

```diff
@@ -174,30 +188,30 @@ app-root {
 //Media query classes
 @include on-med-and-up {
   .pb-hide-on-med-and-up {
-    display: none;
+    display: none !important;
   }
 }
 
 @include below-small {
   .pb-hide-below-small {
-    display: none;
+    display: none !important;
   }
 }
 
 @include below-med {
   .pb-hide-below-med {
-    display: none;
+    display: none !important;
   }
 }
 
 @include below-big {
   .pb-hide-below-big {
-    display: none;
+    display: none !important;
```

Comment:
Do we really need to do it like this @marcuskrahl @DanielHabenicht @Zeroks77 ?

Reply:
I might have a look into it this evening. Couldn't test it with pictures yet.